cmd/do-agent/config.go

```diff
@@ -72,8 +72,8 @@ func init() {
 	kingpin.Flag("syslog", "enable logging to syslog").
 		BoolVar(&config.syslog)
 
-	kingpin.Flag("k8s", "enable DO Kubernetes metrics collection (this must be a DOK8s node)").
-		Default("http://127.0.0.1:8080").
+	kingpin.Flag("k8s-metrics-path", "enable DO Kubernetes metrics collection (this must be a DOK8s metrics endpoint)").
+		Default("http://127.0.0.1:8080/metrics").
```

Comment:
You can kill this default. That URL will never work. Lets just check if it was provided and if it wasn't then skip k8s scraping.

Reply:
done